ci/Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+node(label: 'linux') {
+  sh 'npm run docker-build'
+  sh 'npm run docker-test'
+  sh 'archiveArtifacts artifacts: 'build/*.zip''
```

Comment:
Should remove the `sh` here as `archiveArtifact` is a method just like `sh`. So the result would be:

```groovy
archiveArtifacts artifacts: 'build/*.zip'
```